### ISSUE TITLE
request:  ui.R and server.R example

### ISSUE THREAD

**Issue by benporter** [daattali/shinyforms#16]
please provide an example that uses the ui.R and server.R approach rather than the single file app.R approach.

**Comment by daattali** [comment]
For now, I think the best (and only) is to just have that formInfo list variable available to both ui and server. You could achieve this by defining it in `global.R`, or by putting it in some helper file and `source`-ing that file from both ui and server

**Comment by samuel-bohman** [comment]
Defining the `questions` and `formInfo` lists in `server.R` seems to be working just fine for me. No need for a `global.R`.

**Comment by daattali** [comment]
Yes you can define the lists in server, but the ui also needs it, so it needs to be defined in both files
